auth/user_mgt.go

```diff
@@ -665,15 +671,9 @@ func validateAndFormatMfaSettings(mfaSettings MultiFactorSettings, methodType st
 				return nil, fmt.Errorf("\"uid\" is not supported when adding second factors via \"createUser()\"")
 			}
 		case updateUserMethod:
-			if multiFactorInfo.UID == "" {
-				return nil, fmt.Errorf("the second factor \"uid\" must be a valid non-empty string when adding second factors via \"updateUser()\"")
-			}
 		default:
 			return nil, fmt.Errorf("unsupported methodType: %s", methodType)
 		}
-		if err := validateDisplayName(multiFactorInfo.DisplayName); err != nil {
```

Comment:
Wouldn't we want to include a display name for a (new or existing) mfa option? Any downside to requiring this?

Reply:
the only downside it that it's not really required from the API point of view

Reply:
I think it would be ok to require it on the SDK, since this field helps identify the MFA option easily. @lahirumaramba any thoughts?